src/types/lib/index.ts

```diff
@@ -313,4 +312,6 @@ export type ContractVersion = {
   compiler: CompilerVersion;
 };
 
+export type ChainId = `0x${string}`;
```

Comment:
We could extend this to support existing starknet chainId and custom ones.
```suggestion
type ChainId = "0x534e5f4d41494e" | "0x534e5f5345504f4c4941" | ( `0x${string}` & {})

```

Reply:
I will update the type to reflect this. Thank you @tabaktoni